src/hands.cpp

```diff
@@ -437,6 +437,7 @@ auto isRiichi(const GameState& state, int player, const std::vector<const Mahjon
     if (state.turnNum - state.hands.at(player).riichiRound <= ONEROUNDOFTURNS && state.lastCall < state.hands.at(player).riichiRound) {
```

Comment:
I would make a note of what needs to be changed here on the trello, but I think you shouldn't make these changes in this pr. Or you can just split this off to a seperate smaller cr.